### ISSUE TITLE
Where is the DINO self-supervised framework?

### ISSUE THREAD

**Issue by yfchen97** [clovaai/voxceleb_trainer#155]
Dear authors,

Thanks for this great work. In the paper, you proposed the RawNet3 architecture under the DINO self-supervised learning framework. 
But in the code, i only see the RawNet3 architecture. I don‘t see the code for DINO self-supervised framework including loss function and training strategy.
So, is it that you haven't open sourced the DINO self-supervised framework or I missed it？

Looking forward to receiving your reply. Thanks.

**Comment by Jungjee** [comment]
Hi @yfchen97.
No we haven't open sourced the SSL part yet.

We had plans, however, we postponed it in order to match with another paper of ours entitled`Self-supervised curriculum learning for speaker verification`.

Our implementation is also based on the official DINO repository: https://github.com/facebookresearch/dino.
If you can adapt its dataloader and other necessary parts towards audio, I expect you to achieve similar results using RawNet3 architecture.

**Comment by yfchen97** [comment]
Thank you for reply. I think angleproto loss can be used as a self-supervised learning method, what do you think?

**Comment by yfchen97** [comment]
I am interested in your new work and the paper Self-supervised curriculum learning for speaker verification. By the way, when are you going to open source about the new paper？

在2022-07-11 ***@***.***写道：

Hi @yfchen97.
No we haven't open sourced the SSL part yet.

We had plans, however, we postponed it in order to match with another paper of ours entitledSelf-supervised curriculum learning for speaker verification.

Our implementation is also based on the official DINO repository: https://github.com/facebookresearch/dino.
If you can adapt its dataloader and other necessary parts towards audio, I expect you to achieve similar results using RawNet3 architecture.
